resources/core/charts/console/templates/tracing-microfrontend.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "ui.kyma-project.io/v1alpha1"
```

Comment:
please move to: https://github.com/kyma-project/kyma/tree/master/resources/jeager/templates

Reply:
Done.